products/sle/main.pm

```diff
@@ -1016,7 +1016,7 @@ else {
     }
     elsif (installzdupstep_is_applicable()) {
         # Staging cannot be registered, so Staging cannot be patched before testing upgrades in staging
-        if (!is_staging) {
+        if (!is_staging && (get_var('HDDVERSION', '') !~ /opensuse-/)) {
```

Comment:
This test will only support Versions on development, we will use the newest image so do not need to do fully update. Updated the comments. This is a low priority job for me now since I'm focus on SLE12SP5 now and it is no need for SLE12SP5 do auto migration to LEAP, so if need to support patch_sle to do fully update, this will be late to do update then. Thanks.